Ports/VVVVVV/package.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=VVVVVV
+version=2.3.6
+useconfigure=true
+auth_type=sha256
+files="https://github.com/TerryCavanagh/VVVVVV/archive/refs/tags/${version}.tar.gz VVVVVV-${version}.tar.gz a3366aab9e8462d330044ab1ec63927e9f5c3801c0ed96b24f08c553dcb911e9"
+workdir="${port}-${version}"
+configopts=("-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt" "-DCMAKE_BUILD_TYPE=Release" "-Sdesktop_version")
+depends=("SDL2" "SDL2_mixer")
+icon_file="desktop_version/icon.ico"
+launcher_name="VVVVVV"
+launcher_category="Games"
+launcher_command="/opt/VVVVVV/VVVVVV"
```

Comment:
These are also strings without variable expansions, and should be single-quoted.